taxonomy-editor-frontend/src/constants.js

```diff
@@ -1,21 +1,4 @@
-/**
- * Build the taxonomy api url from ui url
- * @param URL location
- * @returns string
- */
-function taxonomyApiUrlFromUi(location) {
-  let components = location.host.split(".");
-  if (components[0] === "ui") {
-    // we build api url by just replacing ui by api
-    components[0] = "api";
-    return location.protocol + "//" + components.join(".") + "/";
-  } else {
-    // this is a default for simple dev setup
-    return "http://localhost:80/";
-  }
-}
-
-export const API_URL = taxonomyApiUrlFromUi(window.location);
+export const API_URL = process.env.REACT_APP_API_URL;
```

Comment:
No you can't do that, in production the file is static and asking process.env does not work (you are in the user browser), that's why it was done like it was !